### DISCUSSION TITLE
v1.2.8

### DISCUSSION THREAD

**Discussion by snowby666** [snowby666/poe-api-wrapper#60] [Announcements]
New features + update example
**Full Changelog:** [`v1.2.6...v1.2.8`](https://github.com/snowby666/poe-api-wrapper/compare/v1.2.6...v1.2.8)

<hr /><em>This discussion was created from the release <a href='https://github.com/snowby666/poe-api-wrapper/releases/tag/v1.2.8'>v1.2.8</a>.</em>

**Comment by johnd0e** [comment]
Very nice!
A couple of more wishes:
- "Your bots" list, like in web ui
- Ability to Remove bots from "my bot list"
- PoeExample, while searching:
  - ability to return to main menu from any other
  - paginations instead of asking user "Enter the number of bots you want to explore or type all to explore all ..."
  - Graceful handling of these cases
    - Search for a user's bots: wrong user name leads to "Invalid cookie. Please try again." error
    - Add a bot directly: wrong bot name raises ValueError.
  - difference is not clear
    ```
    [4] : Search for a user's bots
    [5] : Search for users and their bots
    ```
